### PR TITLE
Add support for dedicated(v3) ELB service

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

- Add support for dedicated(v3) ELB service.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chengxiangdong** [comment]
/retitle Add support for dedicated(v3) ELB service

File: test/e2e/dedicated_loadbalancer_test.go [hunk 1]
```diff
@@ -0,0 +1,296 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```
Comment (niuzhenguo) [inline test/e2e/dedicated_loadbalancer_test.go:2]: s/2022/2023

Comment (chengxiangdong) [reply test/e2e/dedicated_loadbalancer_test.go]: done

File: pkg/cloudprovider/huaweicloud/dedicatedloadbalancer.go [hunk 2]
```diff
@@ -0,0 +1,852 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```
[the remark this answers is not in the document]
Comment (chengxiangdong) [reply pkg/cloudprovider/huaweicloud/dedicatedloadbalancer.go]: done

File: docs/usage-guide.md [hunk 3]
```diff
@@ -6,11 +6,21 @@ Before running the examples below,
...
+If the annotation in the service is empty, the [Loadbalancer Configuration](./huawei-cloud-controller-manager-configuration.md#loadbalancer-configuration) will be used,
+otherwise use the set value.
+
 ## Service Annotations
 
 * `kubernetes.io/elb.class` Required. Specifies the type of ELB service to use. Values are:
+  
   **shared**: Use the shared load balancer service.
 
+  **performance**: Use the dedicated load balancer service.
```
[the remark this answers is not in the document]
Comment (chengxiangdong) [reply docs/usage-guide.md]: done

File: docs/usage-guide.md [hunk 4]
```diff
@@ -6,11 +6,21 @@ Before running the examples below,
...
+
+* `kubernetes.io/elb.availability-zones` Optional. Specifies the list of AZs where the load balancer can be created.
+  This annotation works with dedicated load balancers (`kubernetes.io/elb.class: performance`),
```
[the remark this answers is not in the document]
Comment (chengxiangdong) [reply docs/usage-guide.md]: done

**Pull request merged**